cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,65 @@
+import { user } from '../support/testData'
+
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
-    
+
+  beforeEach(() => {
+    cy.visit('/login');
   });
+  
 
-  it('', () => {
+  it('should provide an ability to login', () => {
+
+    cy.findByPlaceholder('UserName')
+      .type(user.userName);
+
+    cy.findByPlaceholder('Password')
+      .type(user.password);
     
+    cy.get('#login').click();
+
+    cy.contains(user.userName)
+      .should('be.visible');
+
+    cy.url().should('include', '/profile');
   });
+
+  it('should provide an ability to add a book to the collection', () => {
+    cy.login();
```

Comment:
Create a `cy.login()` command to login with API for better code reusability and maintainability 🔄